docs/installation/centos.md

```diff
@@ -12,153 +12,185 @@ parent = "smn_linux"
 
 Docker is supported on the following versions of CentOS:
 
-- [*CentOS 7 (64-bit)*](#installing-docker-centos-7)
-- [*CentOS 6.5 (64-bit)*](#installing-docker-centos-6.5) or later
+* CentOS 7.X 
+* CentOS 6.5 or higher 
+
+Installation on other binary compatible EL6/EL7 distributions such as Scientific
+Linux might succeed, but Docker does not test or support Docker on these
+distributions.
+
+This page instructs you to install using Docker-managed release packages and
+installation mechanisms. Using these packages ensures you get the latest release
+of Docker. If you wish to install using CentOS-managed packages, consult your
+CentOS documentation.
+
+## Prerequisites
+
+Docker requires a 64-bit installation regardless of your CentOS version. Also,
+your kernel must be 3.10 at minimum. CentOS 7 runs the 3.10 kernel, 6.5 does
+not. We make an exception for CentOS 6.5. To run Docker on
```

Comment:
For the kernel; the current documentation says; http://docs.docker.com/installation/binaries/#check-kernel-dependencies

> A 3.10 Linux kernel is the minimum requirement for Docker. Kernels older than 3.10 lack some of the features required to run Docker containers. These older versions are known to have bugs which cause data loss and frequently panic under certain conditions.

>The latest minor version (3.x.y) of the 3.10 (or a newer maintained version) Linux kernel is recommended. Keeping the kernel up to date with the latest minor version will ensure critical kernel bugs get fixed.

From that perspective, I'd say keep it as-is; 3.10 as minimum

No opinion on the correct wording for 6.x